### PR TITLE
"Memory hardening in Parodus in exit scenarios"

### PR DESCRIPTION
**Pull request by akif1716**
Hi All, Please review the memory hardening patch in Parodus

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -618,6 +618,26 @@ int parseCommandLine(int argc,char **argv,ParodusCfg * cfg)
     return 0;
 }
 
+void free_cfg(ParodusCfg *cfg)
+{
+    if (cfg->mtls_client_cert_path != NULL )
+         free(cfg->mtls_client_cert_path);
```
Comment (shilpa24balaji) [inline src/config.c:624]: Set these pointers to null after free, as pointed by the "Use After free" vulnerability

Comment (sadhyama) [reply src/config.c:624]: yes try to fix security warnings reported .
Please submit PR to master branch as that is the latest branch which is pulled in yocto builds.

**Review by sadhyama — CHANGES_REQUESTED** [review]
Please submit PR to master branch as that is the latest branch which is pulled in yocto builds.

**Comment by shilpa24balaji** [comment]
Merged as part of https://github.com/xmidt-org/parodus/pull/425

**Pull request closed without merging**